### PR TITLE
Add selected style and default title to BitNav Component (#547)

### LINKED ISSUE #547: missing selected Item style and default value for title property in BitNav
- on the fluent demo page when an item gets click, it gets a select style, which BitNav does not
- when target property didn't fill by assumer. must take the name as a default value

### PR DESCRIPTION
**Pull request by zoha-shobbar**
this closes #547

### CODE REVIEW

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Navs/BitNav.razor [hunk 1]
```diff
@@ -62,7 +62,7 @@
             }
 
             <a href="@(navLink.Disabled ? "/" : navLink.Url)"
-               title="@navLink.Title"
+               title=" @(navLink.Title.HasValue()?navLink.Title:navLink.Name)"
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Navs/BitNav.razor:65]: please format the added code correctly

**Review by msynk — CHANGES_REQUESTED** [review]

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Navs/BitNav.razor (hunk 1 above)
Comment (zoha-shobbar) [reply src/Client/Web/Bit.Client.Web.BlazorUI/Components/Navs/BitNav.razor:65]: done

File: src/Client/Web/Bit.Client.Web.BlazorUI/Styles/Fluent/Navs/bit-nav.fluent.scss [hunk 2]
```diff
@@ -25,7 +25,7 @@
     @if $enabled {
         color: $Black;
 
-        &:hover {
+        &:hover, &:focus {
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Styles/Fluent/Navs/bit-nav.fluent.scss:28]: I think this is not correct.

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Navs/BitNav.razor [hunk 3]
```diff
@@ -61,15 +61,13 @@
                 </button>
             }
 
-            @if (navLink.Url.HasValue())
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Navs/BitNav.razor:64]: why this line (the if statement) is removed?

**Review by msynk — CHANGES_REQUESTED** [review]

**Review by msynk — APPROVED** [review]

**Pull request merged**